src/components/PublicHeader.tsx

```diff
@@ -70,10 +72,12 @@ const PublicHeader = ({ canOrganize, user, org  } : PublicHeaderProps) : JSX.Ele
                         </View>
                         <>
                             { canOrganize && (
-                                <NextLink href="https://organize.zetk.in">
-                                    <Button data-testid="organize-button" variant="cta">
-                                        <Msg id="misc.publicHeader.organize"/>
-                                    </Button>
+                                <NextLink href={ `/organize/${memberships[0].organization.id}` }>
```

Comment:
This is risky. Although it's perfectly sane to expect that the `memberships` array is not empty with the way that `DefaultLayout` is using `PublicHeader`, it's not safe to assume that will always be the case for any component.

Add some validation further up the render function to make sure that the array is not empty. This makes even more sense if the calculation of `canOrganize` is moved inside this function as suggested in a previous comment.